cmake/f3dPlugin.cmake

```diff
@@ -118,6 +127,8 @@ The `NAME` argument is required. The arguments are as follows:
 macro(f3d_plugin_build)
   cmake_parse_arguments(F3D_PLUGIN "FORCE_STATIC" "NAME;DESCRIPTION;VERSION" "VTK_MODULES;ADDITIONAL_RPATHS" ${ARGN})
 
+  find_package(VTK 9.0 REQUIRED COMPONENTS ${F3D_PLUGIN_VTK_MODULES})
```

Comment:
F3D_PLUGIN_VTK_MODULES should contains F3D VTKExtensions, or VTK modules ?

Reply:
VTK modules

Reply:
I dont remmeber seeing VTK modules listed on existing plugins

Reply:
https://github.com/f3d-app/f3d/blob/185baa389446f509937b2393341fdc9ee0c66c92/plugins/occt/CMakeLists.txt#L44

Reply:
Thanks ^^